code/game/objects/effects/weeds.dm

```diff
@@ -53,9 +53,9 @@
 
 /obj/effect/alien/weeds/Crossed(atom/movable/AM)
 	. = ..()
-	if(ishuman(AM))
-		var/mob/living/carbon/human/H = AM
-		H.next_move_slowdown += 1
+	if(isxeno(AM))
+		var/mob/living/carbon/xenomorph/X = AM
+		X.next_move_slowdown -= 0.4
```

Comment:
I like the idea of the change it reminds me of zerg, but it looks like when both are on weeds the slowdown delta in favor of xenos on weeds is only 40% what is was, am I reading this correctly?

Reply:
apparently is, which is not so bad tbh, it will help slower castes and faster castes wont win more much, TM would be very neccessary